src/main/cpp/mechanisms/noteManager/decoratormods/noteManager_backupManualPlace_State.cpp

```diff
@@ -65,9 +64,8 @@ bool noteManagerbackupManualPlaceState::AtTarget()
 	return attarget;
 }
 
-bool noteManagerbackupManualPlaceState::IsTransitionCondition ( bool considerGamepadTransitions )
+bool noteManagerbackupManualPlaceState::IsTransitionCondition(bool considerGamepadTransitions)
 {
 	// To get the current state use m_mechanism->GetCurrentState()
-
-	return ( considerGamepadTransitions && TeleopControl::GetInstance()->IsButtonPressed ( TeleopControlFunctions::EXAMPLE_MECH_FORWARD ) );
+	return (considerGamepadTransitions && TeleopControl::GetInstance()->IsButtonPressed(TeleopControlFunctions::MANUAL_MODE) && m_mechanism->m_scoringMode == RobotStateChanges::ScoringMode::Placer);
```

Comment:
Add an OR condition of  (m_mechanism->GetCurrentState() == backupManaulLaunch &&  m_mechanism->m_scoringMode == RobotStateChanges::ScoringMode::Placer)

Reply:
Also update from Main and then replace
m_mechanism->m_scoringMode == RobotStateChanges::ScoringMode::Placer
with
m_mechanism->isPlacerMode()